e2e/process-services/process-attachmentList-actionMenu.e2e.ts

```diff
@@ -115,7 +115,7 @@ describe('Attachment list action menu for processes', () => {
 
         viewerPage.checkFileNameIsDisplayed(pngFile.name);
         viewerPage.clickCloseButton();
-        browser.sleep(10000);
+        browser.sleep(3000);
```

Comment:
do we really need sleep here? I thought we've been using "wait for element" or something like that